microsoft/testsuites/performance/storageperf.py

```diff
@@ -187,6 +189,102 @@ def perf_storage_over_nfs_synthetic_tcp_4k(self, result: TestResult) -> None:
     def perf_storage_over_nfs_synthetic_udp_4k(self, result: TestResult) -> None:
         self._perf_nfs(result, protocol="udp")
 
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test data disk performance.
+        """,
```

Comment:
Please specify the schema and usage of test case in the description.

Reply:
Done with PR : https://github.com/microsoft/lisa/pull/2349